addons/gamification/models/gamification_goal.py

```diff
@@ -213,8 +213,6 @@ def update_goal(self):
 
                 else:
                     field_name = definition.field_id.name
-                    field = Obj._fields.get(field_name)
```

Comment:
and keep this for ... a year?

Reply:
Would say to keep it, as it is not a db constraint, you never know what people end up having as data ... and defensive code does not add much LOC.